task_02/graph.cpp

```diff
@@ -0,0 +1,179 @@
+#include <iostream>
+#include <fstream>
+#include <vector>
+
+struct Vertex{
+	int id = 0;
+	std::vector <int> e_ids;
+};
+
+struct Edge{
+	int id = 0;
+	int start_id = 0;
+	int end_id = 0;
+};
+
+class Graph{
+private:
+	std::vector<Vertex> v_list;
+	std::vector<Edge> e_list;
+	bool find_vertex(int i);
+	bool find_edge(int i);
+public:
+	//Graph();
+	void add_vertex(int i);
+	void add_edge(int i, int first, int second);
+	void vertex_input();
+	void edge_input();
+	void init();
+	void json_output();
+};
+
+void Graph::add_vertex(int i){
+	Vertex tmp;
+	tmp.id = i;
```

Comment:
Лучше стараться никогда не оставлять неинициализированные перменные.
```suggestion
      const Vertex tmp = {.id = i};
```